src/web/pages/App.tsx

```diff
@@ -1,64 +1,39 @@
 import { BrowserRouter as Router, Route, Routes } from 'react-router-dom';
-import HomePage from './HomePage';
 import FinOpsHubsPage from './FinOpsHubsPage';
 import FinOpsWorkbooksPage from './FinOpsWorkbooksPage';
 import GovernanceWorkbookPage from './GovernanceWorkbookPage';
 import AzureOptimizationEnginePage from './AzureOptimizationEnginePage';
 import PowerShellModulePage from './PowerShellModulePage';
 import BicepRegistryModulesPage from './BicepRegistryModulesPage';
 import OpenDataPage from './OpenDataPage';
-import {PowerBIReportsPage} from './PowerBIReportsPage';
-import CostOptimizationWorkbookPage from './CostOptimizationWorkbookPage';
-import {SamplePage} from './SamplePage';
+import { SamplePage } from './SamplePage';
 import FluentUIProvider from '../components/FluentUIProvider'; // Use your FluentUIProvider
 import PowerBIReportsPage from './PowerBIReportsPage';
 import CostOptimizationWorkbookPage from './CostOptimizationWorkbookPage';
-import ToolsLandingPage from './ToolsLandingPage';
-
-import './App.css';
 
 /**
  * The main component of the application.
  * Renders the routes for different pages.
  */
 function App() {
-    return (
-        <FluentUIProvider>
-                <Router>
-                        <Routes>
-                        <Route path="/" element={<SamplePage />} />
-                        <Route path="/bicep-registry" element={<BicepRegistryModulesPage />} />
-                        <Route path="/hubs" element={<FinOpsHubsPage />} />
-                        <Route path="/open-data" element={<OpenDataPage />} />
-                        <Route path="/optimization-engine" element={<AzureOptimizationEnginePage />} />
-                        <Route path="/power-bi" element={<PowerBIReportsPage />} />
-                        <Route path="/powershell" element={<PowerShellModulePage />} />
-                        <Route path="/tools" element={<SamplePage />} />
-                        <Route path="/workbooks" element={<FinOpsWorkbooksPage />} />
-                        <Route path="/workbooks/governance" element={<GovernanceWorkbookPage />} />
-                        <Route path="/workbooks/optimization" element={<CostOptimizationWorkbookPage />} />
-                        </Routes>
-                </Router>
-        </FluentUIProvider>
-    );
-=======
   return (
-
-    <Router>
+    <FluentUIProvider>
+      <Router>
         <Routes>
-          <Route path="/" element={<HomePage />} />
+          <Route path="/" element={<SamplePage />} />
```

Comment:
Why are you removing these 2 pages? We may not need the tools page, so that's probably fine, but we'll definitely have a home page.

Reply:
I'll add the homepage in a later pr. Yes, we have a homepage of course. This is just to correct the mistakes done during merging. There were merge markers left and duplicate content when Arthur merged the different changes.